src/elm/Page/Shop/Editor.elm

```diff
@@ -1148,6 +1148,15 @@ encodeUpdateForm sale form =
         ]
 
 
+getNumericValues : String -> String
+getNumericValues value =
+    value
+        |> String.toList
+        |> List.filter (\x -> Char.isDigit x)
```

Comment:
No need to send a function, you can curry it:

```
|> List.filter Char.isDigit
```

Reply:
Good catch. Fixed